### PR TITLE
chore: add LocationSearchIcon as LocationField end adornment

### PR DESCRIPTION
**Pull request by blushi**
## Description

Closes: regen-network/regen-registry#1099

---

### Author Checklist

_All items are required. Please add a note to the item if the item is not applicable and
please add links to any relevant follow up issues._

I have...

- [ ] provided a link to the relevant issue or specification
- [ ] provided instructions on how to test
- [ ] reviewed "Files changed" and left comments if necessary
- [ ] confirmed all CI checks have passed

### How to test

https://deploy-preview-2081--regen-marketplace.netlify.app/project-pages/0114be22-0614-11ee-93b1-0267c2be097b/location

### Reviewers Checklist

_All items are required. Please add a note if the item is not applicable and please **add
your handle next to the items reviewed if you only reviewed selected items**._

I have...

- [ ] confirmed all author checklist items have been addressed
- [ ] reviewed code correctness and readability
- [ ] verified React components follow DRY principles
- [ ] reviewed documentation is accurate
- [ ] reviewed tests
- [ ] manually tested (if applicable)

### CODE REVIEW

**Comment by blushi** [comment]
@erikalogie you can see this on https://deploy-preview-2081--regen-marketplace.netlify.app/project-pages/0114be22-0614-11ee-93b1-0267c2be097b/location

File: web-components/src/components/icons/LocationSearchIcon.tsx [hunk 1]
```diff
@@ -0,0 +1,30 @@
...
+    <SvgIcon
+      sx={[...sxToArray(sx)]}
+      fill="none"
+      xmlns="http://www.w3.org/2000/svg"
+      {...props}
+    >
+      <path
+        d="M18.2004 20.9914L19.1469 20.1162C19.5524 19.7413 19.5888 19.0981 19.2283 18.6798L14.5274 13.2242C14.1668 12.8058 13.5459 12.7706 13.1404 13.1456L12.1939 14.0208C11.7884 14.3957 11.7519 15.0389 12.1124 15.4572L16.8134 20.9128C17.1739 21.3312 17.7949 21.3664 18.2004 20.9914Z"
+        fill="#D2D5D9"
+        stroke="#D2D5D9"
```
Comment (flagrede) [inline web-components/src/components/icons/LocationSearchIcon.tsx:21]: Using `currentColor` in SVGs, whenever possible, make them more reusable.

**Review by flagrede — APPROVED** [review]
Looks good !

**Comment by erikalogie** [comment]
it should be #D2D5D9 otherwise looks good

**Comment by blushi** [comment]
> it should be #D2D5D9 otherwise looks good

fixed

**Pull request merged**